sklearn/neighbors/tests/test_lof.py

```diff
@@ -126,8 +124,7 @@ def test_lof_precomputed(random_state=42):
     assert_array_almost_equal(pred_X_Y, pred_D_Y)
 
 
-@ignore_warnings(category=DeprecationWarning)
-# contamination changed to 'auto' 0.22
+@pytest.mark.filterwarnings('ignore:default contamination parameter 0.1')
```

Comment:
Should we be giving the version number of scikit-learn where this line should be remove: "# 0.22" at the end of the line?